### PR TITLE
Porting/docs-team-charter.pod: the docs team charter

### PR DESCRIPTION
**Pull request by rjbs**
**This can wait until 5.34.0 to keep life simple.**

### CODE REVIEW

File: Porting/docs-team-charter.pod [hunk 1]
```diff
@@ -0,0 +1,119 @@
+=head1 PERL DOCUMENTATION TEAM CHARTER
```
Comment (Grinnz) [inline Porting/docs-team-charter.pod]: Needs to start with a `=head1 NAME`

**Comment by Grinnz** [comment]
Other test fail seems to be that it is missing from MANIFEST

**Comment by xsawyerx** [comment]
Should this be added to 5.34.0?

File: Porting/docs-team-charter.pod [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+Both the manager and the Steering Council may add volunteers to or remove
+volunteers from the Docs Team.
+
+=head2 Resources
+
+=head3 Mailing list
+
+The Docs Team will maintain a publicly-archived mailing list for discussion of
+its work, and will be the medium of record for recording team plans and
```
Comment (hvds) [inline Porting/docs-team-charter.pod]: s/and will/which will/ - it's the mailing list, not the Docs Team, that is the medium of record.

**Pull request merged**